mne/tests/test_source_estimate.py

```diff
@@ -1002,3 +1002,22 @@ def test_vol_mask():
 
 
 run_tests_if_main()
+
+
+@testing.requires_testing_data
+def test_mixed_sources_plot_surface():
```

Comment:
I would move this test into `mne/viz/tests/test_3d.py` and add the `requires_pysurfer` decorator, there are a few other examples of such tests in this file. Maybe `traits_test` would be needed too.

Reply:
Addressed by commit cd3e9ec Thanks for the feedback.